incubator/wordpress/templates/NOTES.txt

```diff
@@ -0,0 +1,26 @@
+1. Get the WordPress URL by running:
+
+{{- if contains "NodePort" .Values.serviceType }}
+
+  export NODE_PORT=$(kubectl get --namespace {{ .Release.Namespace }} -o jsonpath="{.spec.ports[0].nodePort}" services {{ template "fullname" . }})
+  export NODE_IP=$(kubectl get nodes --namespace {{ .Release.Namespace }} -o jsonpath="{.items[0].status.addresses[0].address}")
+  echo http://$NODE_IP:$NODE_PORT/
+
+{{- else if contains "LoadBalancer" .Values.serviceType }}
+
+  NOTE: It may take a few minutes for the LoadBalancer IP to be available.
+        Watch the status with: 'kubectl get svc -w {{ template "fullname" . }}'
+
+  export SERVICE_IP=$(kubectl get svc --namespace {{ .Release.Namespace }} {{ template "fullname" . }} -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
+  echo http://$SERVICE_IP/
```

Comment:
We actually had this before, but we removed it because some application take some time to "get initialized", which means that the user will see an "initialization message" only in the root URL (in the rest i.e /admin, you will get 404).

In any case, in the case of Wordpress, let's add `/admin` since it initializes fast and the problem is not visible

Thanks for the review.